pkg/remoteexec/ssh/ssh.go

```diff
@@ -157,7 +158,10 @@ func (s *SshExec) ExecCommands(
 			command = "sudo " + command
 		}
 		// Execute command in a shell
-		command = "/bin/bash -c '" + command + "'"
+		command = "/bin/bash -c '" +
+			// Escape single quotes in commands (' -> '\'')
+			strings.Replace(command, "'", "'\\''", -1) +
```

Comment:
Using a string literal here would make it more readable similar to the comment above.